libc/src/__support/FPUtil/FPBits.h

```diff
@@ -127,7 +127,11 @@ template <> struct FPLayout<FPType::IEEE754_Binary128> {
 };
 
 template <> struct FPLayout<FPType::X86_Binary80> {
+#if __SIZEOF_LONG_DOUBLE__ == 16
   using StorageType = UInt128;
+#else
```

Comment:
Thinking ahead to android support, if android uses `long double == double` (https://developer.android.com/ndk/guides/abis#x86) https://godbolt.org/z/e35cEK4fM, then perhaps this aught to be an explicit check for `__SIZEOF_LONG_DOUBLE__ == 12` with an `#else` clause that errors explicitly so that we get a nice build failure if we tried to build llvm-libc using the android triple, and we could investigate proper `long double == double` support.

Reply:
https://github.com/llvm/llvm-project/pull/115084/commits/474aad73d1213340663ceb631ba2c78ef774b9ca